### PR TITLE
Sass 841

### PR DESCRIPTION
**Pull request by BenCosfordHMRC**
https://jira.tools.tax.service.gov.uk/browse/SASS-587

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [x]  Have you addressed warnings where appropriate?

##### After PRs been Reviewed
- [x]  Have you checked the PR Builder passes?
- [x]  Have you checked for merge conflicts?
- [x]  Have you checked code coverage isn’t lower than previously?

To be merged after dependency ticket is merged.

### CODE REVIEW

File: app/views/employment/SingleEmploymentSummaryView.scala.html [hunk 1]
```diff
@@ -77,7 +77,7 @@
 </div>
 }
 
-@layout(pageTitle = headingForUse, taxYear = Some(taxYear), isAgent = user.isAgent) {
+@layout(pageTitle = messages("employment.employmentSummary.heading"), taxYear = Some(taxYear), isAgent = user.isAgent) {
```
Comment (Dula-J) [inline app/views/employment/SingleEmploymentSummaryView.scala.html:80]: Has the content changed from what was demoed last week with this page?

Comment (BenCosfordHMRC) [reply app/views/employment/SingleEmploymentSummaryView.scala.html:80]: Yeah Joey pointed out that the title was different if it was a single employment or multiple so made them the same title, but keeping the heading as is. 

Having the title as the employer name is likely to be a personal information issue as they will be seen on google analytics.

File: it/connectors/IncomeTaxUserDataConnectorSpec.scala [hunk 2]
```diff
@@ -57,6 +58,18 @@ class IncomeTaxUserDataConnectorSpec extends IntegrationTest{
     }
 
     "Return an error result" when {
+
+      "submission returns a 500 when the call is external as the headers won't be passed along" in {
```
Comment (Dula-J) [inline it/connectors/IncomeTaxUserDataConnectorSpec.scala:62]: This test seems to rely on wiremock not being able to match the request due to missing headers, resulting in a 404, which is then handled and converted in an error with InternalServerError

The way the test is written now, the wording should probably be, "Return 500 error when submission returns 404", which I don't think is the intent here.

Comment (BenCosfordHMRC) [reply it/connectors/IncomeTaxUserDataConnectorSpec.scala]: Updated the test name on SASS-687 & merged into here

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,19 +1,17 @@
...
-resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
+resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
+resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.14.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
```
Comment (Dula-J) [inline project/plugins.sbt]: this is no longer required?

Comment (BenCosfordHMRC) [reply project/plugins.sbt]: I believe it's included as part of auto-build 3.0.0

File: it/controllers/employment/EmploymentSummaryControllerISpec.scala [hunk 4]
```diff
@@ -68,6 +68,19 @@ class EmploymentSummaryControllerISpec extends IntegrationTest with ViewHelpers
         welshToggleCheck(ENGLISH)
       }
 
+      "redirect when there is no data" which {
```
Comment (Dula-J) [inline it/controllers/employment/EmploymentSummaryControllerISpec.scala:71]: We have a similar test (Line 189) where IncomeTaxUserData() is completely empty. This test seems to be based on the hmrc segments in the IncomeTaxUserData being empty. Maybe rename this test so the distinction is clearer.

Comment (BenCosfordHMRC) [reply it/controllers/employment/EmploymentSummaryControllerISpec.scala]: Updated to better reflect the scenario

**Pull request merged**